[2022-09-18 18:42:53,476] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: bigquery_dag.write_query_big_query_table 2022-09-15T03:00:00+00:00 [queued]>
[2022-09-18 18:42:53,630] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: bigquery_dag.write_query_big_query_table 2022-09-15T03:00:00+00:00 [queued]>
[2022-09-18 18:42:53,640] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2022-09-18 18:42:53,644] {taskinstance.py:1018} INFO - Starting attempt 1 of 3
[2022-09-18 18:42:53,646] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2022-09-18 18:42:53,730] {taskinstance.py:1038} INFO - Executing <Task(BigQueryOperator): write_query_big_query_table> on 2022-09-15T03:00:00+00:00
[2022-09-18 18:42:53,763] {standard_task_runner.py:51} INFO - Started process 2652 to run task
[2022-09-18 18:42:53,813] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'bigquery_dag', 'write_query_big_query_table', '2022-09-15T03:00:00+00:00', '--job-id', '329', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_bigquery.py', '--cfg-path', '/tmp/tmp866ss35f']
[2022-09-18 18:42:53,836] {standard_task_runner.py:76} INFO - Job 329: Subtask write_query_big_query_table
[2022-09-18 18:42:54,072] {logging_mixin.py:103} INFO - Running <TaskInstance: bigquery_dag.write_query_big_query_table 2022-09-15T03:00:00+00:00 [running]> on host 27377494ab8b
[2022-09-18 18:42:54,337] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Dave
AIRFLOW_CTX_DAG_ID=bigquery_dag
AIRFLOW_CTX_TASK_ID=write_query_big_query_table
AIRFLOW_CTX_EXECUTION_DATE=2022-09-15T03:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-15T03:00:00+00:00
[2022-09-18 18:42:54,353] {bigquery.py:680} INFO - Executing: 
        select * from githubarchive.day.20220914 where actor.id=66800817
            
[2022-09-18 18:42:54,480] {logging_mixin.py:103} WARNING - /home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py:2052 DeprecationWarning: This method is deprecated. Please use `BigQueryHook.insert_job` method.
[2022-09-18 18:42:54,666] {bigquery.py:1510} INFO - Inserting job airflow_1663526574663045_4e8e27051e4b7eda1ea1c14d0ba14e9a
[2022-09-18 18:42:56,208] {taskinstance.py:1396} ERROR - 403 Access Denied: Table githubarchive:day.20220914: User does not have permission to query table githubarchive:day.20220914, or perhaps it does not exist in location northamerica-northeast1.

(job ID: airflow_1663526574663045_4e8e27051e4b7eda1ea1c14d0ba14e9a)

                       -----Query Job SQL Follows-----                       

    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
   1:
   2:        select * from githubarchive.day.20220914 where actor.id=66800817
   3:            
    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1086, in _run_raw_task
    self._prepare_and_execute_task_with_callbacks(context, task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1260, in _prepare_and_execute_task_with_callbacks
    result = self._execute_task(context, task_copy)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1300, in _execute_task
    result = task_copy.execute(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 706, in execute
    encryption_configuration=self.encryption_configuration,
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 2186, in run_query
    job = self.insert_job(configuration=configuration, project_id=self.project_id)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/common/hooks/base_google.py", line 425, in inner_wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 1512, in insert_job
    job.result()
  File "/home/airflow/.local/lib/python3.6/site-packages/google/cloud/bigquery/job/query.py", line 1159, in result
    super(QueryJob, self).result(retry=retry, timeout=timeout)
  File "/home/airflow/.local/lib/python3.6/site-packages/google/cloud/bigquery/job/base.py", line 631, in result
    return super(_AsyncJob, self).result(timeout=timeout, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/google/api_core/future/polling.py", line 134, in result
    raise self._exception
google.api_core.exceptions.Forbidden: 403 Access Denied: Table githubarchive:day.20220914: User does not have permission to query table githubarchive:day.20220914, or perhaps it does not exist in location northamerica-northeast1.

(job ID: airflow_1663526574663045_4e8e27051e4b7eda1ea1c14d0ba14e9a)

                       -----Query Job SQL Follows-----                       

    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
   1:
   2:        select * from githubarchive.day.20220914 where actor.id=66800817
   3:            
    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
[2022-09-18 18:42:56,291] {taskinstance.py:1440} INFO - Marking task as UP_FOR_RETRY. dag_id=bigquery_dag, task_id=write_query_big_query_table, execution_date=20220915T030000, start_date=20220918T184253, end_date=20220918T184256
[2022-09-18 18:42:56,621] {local_task_job.py:118} INFO - Task exited with return code 1
[2022-09-18 19:38:52,854] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: bigquery_dag.write_query_big_query_table 2022-09-15T03:00:00+00:00 [queued]>
[2022-09-18 19:38:52,952] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: bigquery_dag.write_query_big_query_table 2022-09-15T03:00:00+00:00 [queued]>
[2022-09-18 19:38:52,957] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2022-09-18 19:38:52,959] {taskinstance.py:1018} INFO - Starting attempt 1 of 3
[2022-09-18 19:38:52,961] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2022-09-18 19:38:53,050] {taskinstance.py:1038} INFO - Executing <Task(BigQueryOperator): write_query_big_query_table> on 2022-09-15T03:00:00+00:00
[2022-09-18 19:38:53,071] {standard_task_runner.py:51} INFO - Started process 2219 to run task
[2022-09-18 19:38:53,108] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'bigquery_dag', 'write_query_big_query_table', '2022-09-15T03:00:00+00:00', '--job-id', '409', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_bigquery.py', '--cfg-path', '/tmp/tmpsnjs1efv']
[2022-09-18 19:38:53,132] {standard_task_runner.py:76} INFO - Job 409: Subtask write_query_big_query_table
[2022-09-18 19:38:53,492] {logging_mixin.py:103} INFO - Running <TaskInstance: bigquery_dag.write_query_big_query_table 2022-09-15T03:00:00+00:00 [running]> on host 27377494ab8b
[2022-09-18 19:38:53,761] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Dave
AIRFLOW_CTX_DAG_ID=bigquery_dag
AIRFLOW_CTX_TASK_ID=write_query_big_query_table
AIRFLOW_CTX_EXECUTION_DATE=2022-09-15T03:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-15T03:00:00+00:00
[2022-09-18 19:38:53,781] {bigquery.py:680} INFO - Executing: 
        select * from githubarchive.day.20220914 where actor.id=66800817
            
[2022-09-18 19:38:53,888] {logging_mixin.py:103} WARNING - /home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py:2052 DeprecationWarning: This method is deprecated. Please use `BigQueryHook.insert_job` method.
[2022-09-18 19:38:54,097] {bigquery.py:1510} INFO - Inserting job airflow_1663529934091186_4e8e27051e4b7eda1ea1c14d0ba14e9a
[2022-09-18 19:38:55,364] {taskinstance.py:1396} ERROR - 403 Access Denied: Table githubarchive:day.20220914: User does not have permission to query table githubarchive:day.20220914, or perhaps it does not exist in location northamerica-northeast1.

(job ID: airflow_1663529934091186_4e8e27051e4b7eda1ea1c14d0ba14e9a)

                       -----Query Job SQL Follows-----                       

    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
   1:
   2:        select * from githubarchive.day.20220914 where actor.id=66800817
   3:            
    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1086, in _run_raw_task
    self._prepare_and_execute_task_with_callbacks(context, task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1260, in _prepare_and_execute_task_with_callbacks
    result = self._execute_task(context, task_copy)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1300, in _execute_task
    result = task_copy.execute(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 706, in execute
    encryption_configuration=self.encryption_configuration,
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 2186, in run_query
    job = self.insert_job(configuration=configuration, project_id=self.project_id)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/common/hooks/base_google.py", line 425, in inner_wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 1512, in insert_job
    job.result()
  File "/home/airflow/.local/lib/python3.6/site-packages/google/cloud/bigquery/job/query.py", line 1159, in result
    super(QueryJob, self).result(retry=retry, timeout=timeout)
  File "/home/airflow/.local/lib/python3.6/site-packages/google/cloud/bigquery/job/base.py", line 631, in result
    return super(_AsyncJob, self).result(timeout=timeout, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/google/api_core/future/polling.py", line 134, in result
    raise self._exception
google.api_core.exceptions.Forbidden: 403 Access Denied: Table githubarchive:day.20220914: User does not have permission to query table githubarchive:day.20220914, or perhaps it does not exist in location northamerica-northeast1.

(job ID: airflow_1663529934091186_4e8e27051e4b7eda1ea1c14d0ba14e9a)

                       -----Query Job SQL Follows-----                       

    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
   1:
   2:        select * from githubarchive.day.20220914 where actor.id=66800817
   3:            
    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
[2022-09-18 19:38:55,417] {taskinstance.py:1440} INFO - Marking task as UP_FOR_RETRY. dag_id=bigquery_dag, task_id=write_query_big_query_table, execution_date=20220915T030000, start_date=20220918T193852, end_date=20220918T193855
[2022-09-18 19:38:55,684] {local_task_job.py:118} INFO - Task exited with return code 1
